stm/stm.mli

```diff
@@ -66,12 +79,9 @@ type stm_init_options =
   (** [vo] load paths for the document. Usually extracted from -R
      options / _CoqProject *)
 
-  ; require_libs : (string * string option * bool option) list
-  (** Require [require_libs] before the initial state is
-     ready. Parameters follow [Library], that is to say,
-     [lib,prefix,import_export] means require library [lib] from
-     optional [prefix] and [import_export] if [Some false/Some true]
-     is used.  *)
+  ; injections : injection_command list
```

Comment:
Not sure about the name `injections` , I cannot come with a much better one tho; maybe `startup_params` ?

Reply:
I kind of like that "injection" name.